### PR TITLE
mrc-2599 Fix report link on changelog page

### PR DESCRIPTION
**Pull request by mwoodbri**
See mrc-2599 for background. To test, follow the steps to reproduce bug - should fail on master but work with this branch.

### CODE REVIEW

File: src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/templates/VersionPage/ChangelogTabTests.kt [hunk 1]
```diff
@@ -31,7 +31,7 @@ class ChangelogTabTests: BaseVersionPageTests()
         assertThat(doc.select("#changelog-tab p").count()).isEqualTo(0)
 
         val cells = rows[0].select("td")
-        assertThat(cells[0].selectFirst("a").attr("href")).isEqualTo("/reports/r1/v1")
+        assertThat(cells[0].selectFirst("a").attr("href")).isEqualTo("http://localhost:8888/report/r1/v1")
```
Comment (mwoodbri) [inline src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/templates/VersionPage/ChangelogTabTests.kt:34]: viz [IndexTests](https://github.com/vimc/orderly-web/blob/7e37c5e32b27b360c9e189f5610de542e4c47773/src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/templates/IndexTests.kt#L100-L101)

**Review by Nicolas-Dolan — COMMENTED** [review]
I've not been able to replicate this bug. I always get errors when I run reports locally so no link is ever generated that I can see.

**Comment by Nicolas-Dolan** [comment]
After getting the reports to run properly in master, I wasn't able to replicate the bug. Clicking the link brought me to this page. Is this what is expected? 
image

**Comment by mwoodbri** [comment]
> After getting the reports to run properly in master, I wasn't able to replicate the bug. Clicking the link brought me to this page. Is this what is expected?

You're on `master` and clicking the "Tue Sep 07" link here?

Screenshot from 2021-09-07 12-44-01

**Review by EmmaLRussell — APPROVED** [review]
LGTM

**Review by Nicolas-Dolan — APPROVED** [review]
I was able to replicate and the change fixed the bug.

**Pull request merged**